doc/guides/modules.rst

```diff
@@ -433,6 +433,24 @@ module:
      cmake: .
      kconfig: Kconfig
 
+.. _modules_build_files_extern:
+
+Build files in a module root
+============================
+
+Inclusion of build files located outside of the Zephyr module itself, can be
+described as:
+
+.. code-block:: yaml
+
+   build:
+     cmake-extern: True
+     kconfig-extern: True
+
+This allows control of the build inclusion to be described externally to the
+Zephyr module.
+
+The Zephyr repository itself is always added as a Zephyr module root.
```

Comment:
I suggest pointing to the section about module roots here.

Reply:
Good point, will do.

Reply:
section re-wriiten and moved.

Link added in beginning of the section.